### ISSUE TITLE
A11Y_Oracle Data Migration extension for Azure Data Studio_data migration oracle_Summary_Screen Reader: Screen Reader is not announcing "Link 1" label name.

### ISSUE THREAD

**Issue by PadminiGariki** [microsoft/azuredatastudio#24928]
### GitHub Tags:
#A11yWCAG2.1; #A11yTCS; #A11ySev2; #DesktopApp; #Benchmark; #BM-AzureDataStudio-Win32-Feb2022; #Azure Data Studio; #Narrator; #WCAG1.3.1; #Info and Relationships;#STP; #FTP; #ScreenReader; #Win11; #Win32; #NVDA;#Element:link;

### Environment Details:
Application Name: Oracle Data Migration extension for Azure Data Studio
Download application: Azure Data Studio.
Version:1.45.1(system setup)
Operating System: Window 11 version 22H2 OS build 25267.1000.
Screen Reader: NVDA Version 2023.1(2023.1.0.27913)
### Repro Steps:

1. Open Azure Data Studio application.
2. Install "Data Migration for Oracle"
3. connect to source connection.
4. Tab till new Migration and invoke on it.
5. Tab till step-6 "Summary" and invoke on it.
6. Now enable narrator.
7. Tab till "Link 1"
8. Now verify that Narrator is not announcing "Link 1" label name.

### Actual Result:
 Screen Reader is not announcing "Link 1" label name.
**Note:**
Similar issue observed with NVDA.
### Expected Result:
 Screen Reader should announce "Link 1" label name.
### User Impact:
Screen Reader users will not be able to know "Link 1" label name. If narrator is not announced.
### Attachment:
![2663188_codesnippet](https://github.com/microsoft/azuredatastudio/assets/142479541/d249f8d0-b1fb-47bc-b17b-5f5fd87518f7)

https://github.com/microsoft/azuredatastudio/assets/142479541/471f8eb8-f7e8-4a72-b570-f5d35015ac10

https://github.com/microsoft/azuredatastudio/assets/142479541/92256db8-062d-4f8b-a7fe-601422d52e84

**Comment by PadminiGariki** [comment]
primary bug id: https://msdata.visualstudio.com/Migration%20Assistant/_workitems/edit/2663188

**Comment by Swati1700** [comment]
@kisantia Could you please provide the ETA of fix. When the fixes will be available in the environment.

**Comment by kisantia** [comment]
@SrivenNookala @Bhavikshah123 please help provide an ETA. Thanks!

**Comment by PadminiGariki** [comment]
@kisantia 
Verified the issue in below environment.
Application Name: Oracle Data Migration extension for Azure Data Studio
Download application: Azure Data Studio.
Version: 1.47.1 (user setup)
Operating System: Window 11 version 23H2 
Screen Reader: NVDA Version 2023.3, Narrator.

issue is fixed. Hence the bug is Closed.
#Closed; #Regressed:02-29-24;
For reference, attachment is added.

Attachments

https://github.com/microsoft/azuredatastudio/assets/142479541/ed146062-2fde-425f-9692-128818e9a968

https://github.com/microsoft/azuredatastudio/assets/142479541/c3e96d7c-d453-4ed0-a06a-6861b54d17fe

**Comment by kisantia** [comment]
closing as confirmed fixed above
